app/controllers/comments_controller.rb

```diff
@@ -232,10 +232,32 @@ def unhide
     end
   end
 
+  def admin_delete
+    @comment = Comment.find(params[:comment_id])
+    authorize @comment
+    @comment.deleted = true
+
+    if @comment.save
+      redirect_url = @comment.commentable&.path
+      if redirect_url
+        redirect_to redirect_url, notice: "Comment was successfully deleted."
```

Comment:
My `notice` and `alert` are STILL not firing. To be clear, I'm expecting a banner to appear at the top of the page, with the specified message, after the redirect happens.

Reply:
I think what you want here is
```
flash[:success] = "Comment was successfully deleted."
```

Then for failures
```
flash[:danger] = "Something went wrong..."
```

Reply:
🙏🏽

Reply:
Doesn't seem to be working Molly. I looked into the mechanism for the messages confirming changes under Settings, but I do not think it uses the same approach as here.

Reply:
I think I have it now 👍